JSTests/test262/config.yaml

```diff
@@ -173,42 +159,7 @@ skip:
-    - test/built-ins/Temporal/Duration/prototype/total/precision-exact-mathematical-values-5.js
-    - test/built-ins/Temporal/Duration/prototype/total/throws-on-wrong-offset-for-zoneddatetime-relativeto.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/rounding-relative.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingincrement.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-ceil.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-expand.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-floor.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-halfCeil.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-halfEven.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-halfExpand.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-halfFloor.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-halfTrunc.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-trunc.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-undefined.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/roundingmode-wrong-type.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/smallestunit-higher-units.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/smallestunit-plurals-accepted.js
-    - test/built-ins/Temporal/PlainDate/prototype/since/smallestunit-wrong-type.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/rounding-relative.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingincrement.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-ceil.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-expand.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-floor.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-halfCeil.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-halfEven.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-halfExpand.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-halfFloor.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-halfTrunc.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-trunc.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-undefined.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/roundingmode-wrong-type.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/smallestunit-higher-units.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/smallestunit-plurals-accepted.js
-    - test/built-ins/Temporal/PlainDate/prototype/until/smallestunit-wrong-type.js
+    - test/built-ins/Temporal/Duration/prototype/total/unit-plurals-accepted.js
```

Comment:
This line seems like it was moved from line 319 below.

Reply:
That was on purpose -- it depends on the `relativeTo` option, but not on `PlainMonthDay/PlainYearMonth`.